protoc-gen-swagger/genswagger/template.go

```diff
@@ -0,0 +1,432 @@
+package genswagger
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+
+	//"github.com/golang/glog"
+
+	"github.com/gengo/grpc-gateway/protoc-gen-grpc-gateway/descriptor"
+	pbdescriptor "github.com/golang/protobuf/protoc-gen-go/descriptor"
+)
+
+// Discover all possible enums that can be generated and insert them in the
```

Comment:
It is better to make the comment a full sentence(s) to follow the standard godoc convention, although this comment is not extracted by default.